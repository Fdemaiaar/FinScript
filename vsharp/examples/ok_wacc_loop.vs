var e = 300 ;
var d = 200 ;
var ce = 0.12 ;
var cd = 0.07 ;
var i = 1 ;
while (i <= 3) {
  print("WACC", i , "=", wacc(e , d , ce , cd)) ;
  i = i + 1 ;
}
