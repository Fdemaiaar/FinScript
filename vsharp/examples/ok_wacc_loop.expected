WACC 1 = 0.101538
WACC 2 = 0.101538
WACC 3 = 0.101538
